### ISSUE TITLE
Sending status code is not working

### ISSUE THREAD

**Issue by JCMais** [colyseus/uWebSockets-express#12]
Hi, first, thanks for this library! It is really helpful.

I found a possible issue when using the library, it seems that a route like this one:

```
    app.get('/test', (_req, res) => {
        return res.status(401).send()
    })
```

Is not sending the `401` status to the client.

I found this when trying to use this with `express-basic-auth`.

Using `res.res.writeStatus(401).end()` does work, but that means accessing the internal uWS Response object.

Another possible issue is that handlers created with `router.post` are also handling `get` requests:

```
router.get('/subpath', ...) // this also handles post requests - not just get
app.use('/path', router)
```

**Comment by endel** [comment]
Just published `uwebsockets-express@1.2.0` with these changes https://github.com/colyseus/uWebSockets-express/pull/13 (now the built-in express router is being used rather than being mocked/mimicked)

**Comment by JCMais** [comment]
Thanks for getting this fixed @endel! 🚀
